### ISSUE TITLE
Pre-build images

### ISSUE THREAD

**Issue by tux2000** [snowzach/hassio-addons#32]
I realized that my home assistant backups contain the full docker image for doods2. I suspect that this is due to the addon being build locally. Would it be possible/sensible to publish pre-build images in a container repository?

https://developers.home-assistant.io/docs/add-ons/publishing

Did not yet figure out what is already available, but there seem to exist some helper actions as e.g. used in the signal addon deploy:
https://github.com/haberda/signal-addon/blob/main/.github/workflows/deploy.yaml

**Comment by snowzach** [comment]
I really don't understand how it's supposed to work. It's bee a real long time since I did this but if I recall, basically if you want it to work for more than 1 architecture, you have to build it and specify the image per architecture. 

There's already a multi-arch image that exists `snowzach/doods2:latest` that should cover it.. I dunno.. The addons thing never made a lot of sense to me.

**Comment by tux2000** [comment]
Yeah, i find it quite confusing too, but there is an {arch} wildcard in the image field and the github workflow builds 3 architecture specific images based on your upstream images.
I made some more changes on my fork to be able to use it as installation source. Will test it from there for a while to see if this will remove the extra 1GB from my daily backups...
